### PR TITLE
Taxonomy stuff

### PR DESCRIPTION
**Pull request by Singletoned**

### CODE REVIEW

File: skills_review/skills/management/commands/initialise_taxonomy.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
+
+from skills_review.skills import taxonomy
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        taxonomy.create_job_title_embeddings()
+        print("Initialised")
```
Comment (252afh) [inline skills_review/skills/management/commands/initialise_taxonomy.py]: We could be more explicit here for what's been initialised - to futureproof against more commands being added without it being ambiguous with what's happened.

File: skills_review/skills/taxonomy.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    skill_count = 10
+    model_name = "all-MiniLM-L6-v2"
+    missing_ratings = False
+
+    if job_title not in job_embeddings.index.to_list():
+        user_skills = user_skills[user_skills["job_title"] != job_title].copy()
+
+    unique_job_titles = user_skills.drop_duplicates(subset=["job_title"]).dropna(axis=0)["job_title"].to_numpy()
+
+    if missing_ratings:
```
Comment (252afh) [inline skills_review/skills/taxonomy.py]: Can this ever be true?

File: skills_review/templates/index.pug [hunk 3]
```diff
@@ -7,4 +7,5 @@ block content
       form(method="POST")
         {{csrf_input}}
 
+        <input type="hidden" name="source" value="{{source|default('openai')}}">
```
Comment (252afh) [inline skills_review/templates/index.pug:10]: How does the user actually switch between openAI and taxonomy sources? Or is this a system setting we're using?

**Review by 252afh — APPROVED** [review]

File: skills_review/skills/taxonomy.py (hunk 2 above)
Comment (Singletoned) [reply skills_review/skills/taxonomy.py]: Just to be clear.  That was Andreas' code not mine

File: skills_review/templates/index.pug (hunk 3 above)
Comment (252afh) [reply skills_review/templates/index.pug:10]: Offline convo: URL matches switch between openai and taxonomy

File: skills_review/skills/management/commands/initialise_taxonomy.py (hunk 1 above)
Comment (Singletoned) [reply skills_review/skills/management/commands/initialise_taxonomy.py]: Made it slightly more explicit

File: skills_review/templates/index.pug (hunk 3 above)
Comment (Singletoned) [reply skills_review/templates/index.pug:10]: They just go to a different url (`/taxonomy`) and it's all handled for them

**Pull request merged**